pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -272,6 +272,8 @@ var _ = Describe("v1beta1 inference service controller", func() {
 								FSGroupChangePolicy: nil,
 								SeccompProfile:      nil,
 							},
+							ServiceAccountName:       constants.KserveServiceAccountName,
```

Comment:
I'm not seeing right to change the default service account. It could be considered backwards incompatible.